### PR TITLE
v2

### PR DESCRIPTION
**Pull request by phamann**
This includes all of the discussed changes for v2 of the API. We've decided to bump to v2 and the new interface is essentially a breaking change.

The biggest change is all relationship requests now **must** include a type, such as:
`/v2/user/111-222-333/followed/concept`

This will allow us to have multiple relationships with the same label pointing to different nodes, such as:
`(List)-[:contains]->(Content)`
`(List)-[:contains]->(Page)`
`(List)-[:contains]->(Concept)`

All of the important changes are at the top of the PR, the bottom files are just the tests copied over.

/cc @Financial-Times/next-myft

### CODE REVIEW

**Comment by wheresrhys** [comment]
All looks pretty clean :+1: 
- update README
- update '/' to redirect to v2 docs

cc @rowanbeentje you'll need to move over to v2 of myft-api (changes are quite minimal)

File: tests/v2/reationship.spec.js [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+					request(api)
+						.get('/v2/user/' + userUuid + '/followed/concept/' + 'TnN0ZWluX09OX0ZvcnR1bmVDb21wYW55X0FBUEw=-T04=')
+						.end(function(err, res){
+							expect(res.status).to.equal(404);
+							done();
+						});
+				});
+		});
+
+		it('DELETE should delete a given inward relationship', (done) => {
```
Comment (adgad) [inline tests/v2/reationship.spec.js]: Are we still keeping this inward DELETEs? (Isn't being used anywhere and possibly wont be?)

**Comment by phamann** [comment]
> update '/' to redirect to v2 docs

Yes, but I haven't even finished the `v1` docs yet :stuck_out_tongue_closed_eyes: 

Will do whilst away next week.

File: tests/v2/reationship.spec.js (hunk 1 above)
Comment (phamann) [inline tests/v2/reationship.spec.js]: I don't see the harm?

**Pull request merged**